src/pybind/mgr/cephadm/templates/services/nfs/ganesha.conf.j2

```diff
@@ -2,7 +2,8 @@
 NFS_CORE_PARAM {
         Enable_NLM = false;
         Enable_RQUOTA = false;
-        Protocols = 4;
+        Protocols = 3, 4;
```

Comment:
still missing one necessary  test change related to this
```
2024-09-18T05:50:29.842 INFO:tasks.cephfs_test_runner:Traceback (most recent call last):
2024-09-18T05:50:29.842 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_bfaa8e3142d35bc978916c2ddbe9ef8b4aceb53f/qa/tasks/cephfs/test_nfs.py", line 470, in test_create_and_delete_export
2024-09-18T05:50:29.842 INFO:tasks.cephfs_test_runner:    self._test_get_export()
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:  File "/home/teuthworker/src/git.ceph.com_ceph-c_bfaa8e3142d35bc978916c2ddbe9ef8b4aceb53f/qa/tasks/cephfs/test_nfs.py", line 291, in _test_get_export
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:    self.assertDictEqual(self.sample_export, nfs_output)
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:AssertionError: {'export_id': 1, 'path': '/', 'cluster_id':[257 chars]: []} != {'access_type': 'RW', 'clients': [], 'clust[260 chars]CP']}
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:  {'access_type': 'RW',
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:   'clients': [],
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:   'cluster_id': 'test',
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:   'export_id': 1,
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:   'fsal': {'cmount_path': '/',
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:            'fs_name': 'nfs-cephfs',
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:            'name': 'CEPH',
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:            'user_id': 'nfs.test.nfs-cephfs.3746f603'},
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:   'path': '/',
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:-  'protocols': [4],
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:+  'protocols': [3, 4],
2024-09-18T05:50:29.843 INFO:tasks.cephfs_test_runner:?                +++
2024-09-18T05:50:29.844 INFO:tasks.cephfs_test_runner:
2024-09-18T05:50:29.844 INFO:tasks.cephfs_test_runner:   'pseudo': '/cephfs',
2024-09-18T05:50:29.844 INFO:tasks.cephfs_test_runner:   'security_label': True,
2024-09-18T05:50:29.844 INFO:tasks.cephfs_test_runner:   'squash': 'none',
2024-09-18T05:50:29.844 INFO:tasks.cephfs_test_runner:   'transports': ['TCP']}
2024-09-18T05:50:29.844 INFO:tasks.cephfs_test_runner:
```
https://pulpito.ceph.com/adking-2024-09-18_05:05:01-orch:cephadm-wip-adk-testing-2024-09-17-1139-distro-default-smithi/7910098/